compute_sdk/globus_compute_sdk/sdk/utils/__init__.py

```diff
@@ -116,3 +116,15 @@ def get_env_var_with_deprecation(
         )
 
     return os.getenv(current_name, os.getenv(old_name, default))
+
+
+def display_name(name: str):
+    """
+    Used for descriptions of diagnostic commands
```

Comment:
Since this now exists in `utils.py`, I think the docstring should reflect is general applicability, rather than explicitly describing its use for "diagnostic commands."